src/scripts/main.js

```diff
@@ -22,7 +22,23 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const thisUl = document.createElement('ul');
+
+  element.append(thisUl);
+
+  for (const item in data) {
+    const newItem = document.createElement('li');
+
+    newItem.innerHTML = item;
+
+    thisUl.append(newItem);
+
+    const a = data[item];
```

Comment:
```suggestion
    const a = data[item];
```
Не дуже інтуїтивна назва для змінної